pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+filterwarnings =
+    ignore::ImportWarning
+    ignore::ResourceWarning
```

Comment:
we now have pytest setting split across `setup.cfg` and `pytest.ini`, can this be moved to `setup.cfg`?